src/classes/CanvasRenderingContext2D.js

```diff
@@ -97,7 +78,7 @@ export default class CanvasRenderingContext2D {
   }
 
   _directionStack = ['inherit'];
-  _fillStyleStack = ['#000'];
```

Comment:
Please don't use MooColor objects. Just use the strings like before.

Nit: Also, the code style for `jest-canvas-mock` is single quoted strings.

Reply:
fixed this now 👍🏻 They get added to the stack as 6-digit hex colors and then re-parsed into a `MooColor` object for serialization